mono/mini/interp/interp.c

```diff
@@ -2036,6 +2037,34 @@ do_transform_method (InterpFrame *frame, ThreadContext *context)
 	interp_pop_lmf (&ext);
 }
 
+#define ALIGN_PTR_TO(ptr,align) (gpointer)((((gssize)(ptr)) + (align - 1)) & (~(align - 1)))
```

Comment:
not really related to this PR: we have this kind of macro all over the codebase, it would be really nice to have a single one...